### ISSUE TITLE
Programatically Cancel event deletion

### ISSUE THREAD

**Issue by vip9** [antoniandre/vue-cal-v4#298]
It is issue/bug or feature request.
I am using vue bootstrap+latest vue-cal. When I click&hold an event, "Delete" slides out.
Normally, I could click elsewhere and "cancel" it (hide the deletion link). 
But I am observing something different: clicking elsewhere does not hide "Delete link". 
Also, clicking on other event, moves "delete link" to that other event.
So, is there any way to detect when user requested deleting event? And/Or programatically cancel it?

**Comment by giulianimartini** [comment]
Hi @vip9,

can you give a codepen to this issue?
I was trying in your another question codepen (https://codepen.io/vipw/pen/zYrBByO) and it is working well.

Give us some code, then we can help :D

**Comment by vip9** [comment]
The problem is with prefs, e.g.:
    :editable-events="{ title: false, drag: false, resize: false, delete: true, create: true }"
causes the issue.
Codepen updated.

**Comment by giulianimartini** [comment]
Hi @vip9,

Yep, indeed it causes the issue.

Can you set these preferences in your logic code? Like a controller or whatever?

I had some similar issues when trying to use the prefs directly in the vue-cal instance, so I set them in my controller.

Example:

```
$events['resizable'] = false;
$events['titleEditable'] = false;
$events['draggable'] = false;
```

Hope it helps.

**Comment by antoniandre** [comment]
Thanks @giulianimartini for you help.

Hi @vip9, I could reproduce the issue, thanks for reporting, this is a bug.
Will let you know when fixed.

**Comment by antoniandre** [comment]
HI @vip9,

This is also a regression involving the same piece of code in the codebase as #299, only happening with editableEvent.resize = false.

Should now be fixed in version 3.5.4, your Codepen is fixed.

Please reopen if not solved, thanks for reporting!
